packages/@lwc/features/src/flags.ts

```diff
@@ -79,4 +79,6 @@ export function setFeatureFlagForTest(name: FeatureFlagName, value: FeatureFlagV
     }
 }
 
+export const runtimeFlags = lwcRuntimeFlags; // backwards compatibility for before this was renamed
```

Comment:
This is good. Just curious what usecase will require this backwards compatibility. Does anybody use `runtimeFlags` from `@lwc/features` outside of our repos?

Reply:
I'm not aware of any, but since this is a public package on npm that (in theory) anyone could use, and since the fix was a one-liner, I added it in. We could definitely remove it in a breaking release. I'll add that to the wishlist: https://github.com/salesforce/lwc/issues/2964